src/main/java/org/kainos/ea/resources/AuthController.java

```diff
@@ -0,0 +1,55 @@
+package org.kainos.ea.resources;
+
+import io.dropwizard.auth.Auth;
+import io.swagger.annotations.Api;
+import org.kainos.ea.api.AuthService;
+import org.kainos.ea.cli.Credential;
+import org.kainos.ea.client.AuthenticationException;
+import org.kainos.ea.client.GenericActionFailedException;
+import org.kainos.ea.db.AuthDao;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api("So SOLID crew Auth API")
+@Path("/api")
+public class AuthController {
+    private AuthService authService = new AuthService(new AuthDao());
+
+    @POST
+    @Path("/login")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response login(Credential login){
+        try {
+            String token = authService.login(login);
+            if(token == null){
+                return Response.status(Response.Status.BAD_REQUEST).build();
+            }
```

Comment:
move this logic to the service, throw an exception that can be caught below. also, I would have assumed the token being null was a problem on our end so should return 500?